indico/queries/workflow.py

```diff
@@ -278,9 +322,21 @@ class WorkflowSubmissionDetailed(WorkflowSubmission):
     detailed_response = True
 
     def __init__(
-        self, workflow_id: int, files: List[str] = None, urls: List[str] = None
+        self,
```

Comment:
Did the black formatter let this happen? I like this style but the formatter used to just change this to all be on one line.

Reply:
yea, it formatted it - not sure why but thats what it decided was best